src/mvba/abba/tests.rs

```diff
@@ -175,18 +174,17 @@ fn test_should_publish_main_vote_message() {
 }
 
 #[test]
-fn test_ignore_messages_with_wrong_id() {
+fn test_ignore_messages_with_wrong_tag() {
     let i = TestNet::PARTY_X;
     let j = TestNet::PARTY_X;
     let mut t = TestNet::new(i, j);
 
     let just = PreVoteJustification::FirstRoundOne(t.c_final.clone());
     let mut pre_vote_x = t.make_pre_vote_msg(1, PreVoteValue::One, &just, &TestNet::PARTY_B);
-    pre_vote_x.id = "another-id".to_string();
+    pre_vote_x.tag = Tag::new("bad tag", 0, 0);
```

Comment:
Here is an example of tag with redundant information. It should be `round` and `i`.